WTWSMS/decisions/WtWSMS_diploaction_decisions.txt

```diff
@@ -96,4 +96,84 @@ targetted_decisions = {
 			factor = 1
 		}
 	}
+	
+	# TODO: ask_to_become_foederati_emperor  (Empire->barbarians)
+	
+	become_foederati_barbarian = { # barbarians->Empire
+		only_independent = yes
+		ai_check_interval = 12
+
+		potential = {
+			OR = {
+				has_landed_title = e_byzantium
+				has_landed_title = e_wre
+				has_landed_title = e_roman_empire
+			}
+		}
```

Comment:
I don't think the foederati should be limited to the Roman empire itself, given that Soissons had such relations as well. Personally I think this could model that: 
```
AND = {
	primary_title = { higher_tier_than = DUKE }
	has_romance_culture_trigger = yes
}

```

Reply:
Done.